lte/gateway/c/session_manager/LocalEnforcer.cpp

```diff
@@ -1148,8 +1148,16 @@ void LocalEnforcer::update_charging_credits(
       continue;
     }
     for (const auto& session : it->second) {
-      std::string sid                = session->get_session_id();
-      SessionStateUpdateCriteria& uc = session_update[imsi][sid];
+      std::string session_id = session->get_session_id();
+      if (session_id != credit_update_resp.session_id()){
+        MLOG(MDEBUG) <<
```

Comment:
This log might get very noisy but I suppose that is what debug is for :) We can remove it once it is stable. It is probably more useful to log the actual update case, which might already exist.

Reply:
Yes i agree.  This won't be noisy until we have multi bearers tests. And it may be useful at the beginning to understand the flow at the beggining when we start to test more bearers.  Once we are confident we can remove it